queries.sql

```diff
@@ -30,7 +30,37 @@ SELECT * FROM animals WHERE NOT name='Gabumon';
 SELECT * FROM animals WHERE (weight_kg BETWEEN 10.4 and 17.3 AND weight_kg =10.4 OR weight_kg= 17.3);
 
 
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
 
+SAVEPOINT SPA1;
 
+ROLLBACK TO SPA1;
+
+UPDATE animals SET weight_kg=weight_kg * -1;
+
+UPDATE animals SET weight_kg=weight_kg * -1 WHERE weight_kg<0;
+
+BEGIN TRANSACTION;
+
+COMMIT;
```

Comment:
- [ ] Please make these transactions in the order that was requested in the project description. Here, you rolled back to SPA1 and after you updated the animals table which is not correct. It should be in this order:
```
    BEGIN;

    DELETE FROM animals WHERE  date_of_birth >= '2022/01/01' ;
    SAVEPOINT SP1;

    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP1;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 

    COMMIT;

```

- [ ] You are also missing the following transactions from the project requirements.
![image](https://user-images.githubusercontent.com/45460192/206211413-ce770540-5fa9-47b1-a8e0-0d40f5c14e06.png)